packages/suite-desktop-core/src/modules/trezor-connect.ts

```diff
@@ -40,6 +41,21 @@ const emitOnSetCustomBackendToMainThreadToAllowDomains = ({
     }
 };
 
+// override TrezorConnect.init and TrezorConnect.setTransports params
+// add BluetoothTransport if bluetooth module is enabled
+const getTransportsParam = (
+    transports?: ConnectSettings['transports'],
+): ConnectSettings['transports'] => {
+    const bluetooth = bluetoothModuleState.getTransport();
+    if (!bluetooth) return transports;
+
+    if (transports && transports.length > 0) {
+        return [...transports, bluetooth];
+    }
+
+    return [bluetooth];
```

Comment:
I have a suspicion that when no transport is explicitly passed, this added `bluetooth` may break implicit fallback to `BridgeTransport` (see https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/device/TransportList.ts#L70)